src/Domain/Donation.php

```diff
@@ -13,11 +14,41 @@
 class Donation {
 	use FreezableValueObject;
 
+	// status for direct debit
+	const STATUS_NEW = 'N';
+
+	// status for bank transfer
+	const STATUS_PROMISE = 'Z';
+
+	// statuses for external payments
+	const STATUS_EXTERNAL_INCOMPLETE = 'X';
+	const STATUS_EXTERNAL_BOOKED = 'B';
+
+	const STATUS_MODERATION = 'P';
+	const STATUS_DELETED = 'D';
+
+	private $status;
 	private $amount;
 	private $interval = 0;
 	private $paymentType;
 	private $personalInfo;
+	private $optIn;
```

Comment:
Yeah, not very descriptive. This indeed is the `info` HTTP request parameter and `info` db field. So this should be a boolean, not a string